### ISSUE TITLE
Lyrics don't display for file names that have [ 

### ISSUE THREAD

**Issue by zoultrias** [clementine-player/Clementine#7117]
### Before posting

Please follow the steps below and check the boxes with [x] once you did the step.

- [ ] I checked the issue tracker for similar issues
- [ ] I checked the [changelog](https://github.com/clementine-player/Clementine/blob/master/Changelog) if the issue is already resolved
- [ ] I tried the latest Clementine build from [here](https://builds.clementine-player.org/)

### System information

Please provide information about your system and the version of Clementine used.

- Operating System:
- Clementine version:

**Comment by lumip** [comment]
I have tried this and it seems to be working for me. Can you give more information of your specific case to reproduce this bug?
- What file format were you using?
- Does this affect lyrics from tags or fetched from the internet? If the latter, which lyrics provider?
- Anything else that might be useful

**Comment by zoultrias** [comment]
What file format were you using?  MP3
Does this affect lyrics from tags or fetched from the internet? If the latter, which lyrics provider? Default lyrics.com
What operating system are you running? W11
Which version of Clementine does this affect? Latest
Anything else that might be useful

The MP3 was likely generated from a website that ripped it from the youtube video. 
Original name was something like this : 
Ed Sheeran - Bad Habits[Official Video].mp3

When I removed the "[Official Video]" portion it started working normally. Now when I add it back in, it doesn't seem to bug out. However, I have several other songs that have "[Mp3Mp4i.online]" at the end, probably same situation. Maybe it has nothing to do with the [ ] symbols like I originally thought. Even when I rename this one and remove "[Mp3Mp4i.online]" it still doesn't seem to update the left pane. Maybe something else in the file, like ID3v2 is corrupted?  How do I check that ?

**Comment by lumip** [comment]
Based on that I would assume that probably there were no ID3 tags present in the file. In that case Clementine tries to guess them from the file name/path, so it ends up with interpret "Ed Sheeran" and title "Bad Habits[Official Video]" in this case, which it then uses to look up lyrics and find no match.
You can check the tags by right-clicking the file in Clementine and then choosing "edit metadata".

If this is indeed what happened, I don't think there's all that much we can do to fix this from our side.

**Comment by zoultrias** [comment]
What you described is what I expected to happen. However it doesn't search for "Bad Habits[Official Video]". When I click the search it still searches for the last valid file that it could load the left pane for. I can't narrow down the cause for now I suppose ticket can't be closed.

**Comment by lumip** [comment]
Not sure if I understood correctly: Does it display the lyrics (and other song info) for the previously played song instead of the "Bad Habits" for you?

**Comment by zoultrias** [comment]
Yes it stays stuck on the lyrics for the previous song. When I click search it clearly tries to search for the previous song's name not the current song being played.

**Comment by sukrosono** [comment]
perhaps we need a log file. well, I know nothing how clementine logging works.
